LEAF_Nexus/sources/Group.php

```diff
@@ -417,6 +430,9 @@ public function listGroupPositions($groupID)
      */
     public function listGroupEmployees($groupID)
     {
+        if(!is_numeric($groupID)){
+            return array();
```

Comment:
Since `Exceptions` are thrown elsewhere when the input is invalid, instead of returning an empty array, it should throw `new Exception('invalid group');`